src/pages/overview/document-generation-api/templatetags.md

```diff
@@ -409,8 +409,36 @@ Here is the list of [supported aggregation functions](https://docs.jsonata.org/a
 aggregate numerical calculation can only be applied to a list of
 numbers.
 
-## Adobe Sign
+## Autogen Marker
+Autogen Marker/Tags represents category of Tags whose values are automatically generated by MS Office, and are not derived from input data. They can be of different types like Table of Content, DateTime Field etc.
+
```

Comment:
Please use "table-of-contents", not "content". Also, what other tags are supported?

Reply:
Hi @cfjedimaster I corrected the name. Its _table-of-contents_ now. Also we are not introducing any more new tags apart from this one.

Reply:
Ok, then I'd change the text to not say there can be different types of tags. If we are only launching with one (TOC), I'd just mention that.

Reply:
ok